app/controllers/tasks/months_controller.rb

```diff
@@ -1,9 +1,21 @@
 class Tasks::MonthsController < ApplicationController
   before_action :verify_user, :validate_uri
+  before_action :set_monthly_tasks, only: %i(index)
```

Comment:
`before_action` is before action. it has too much process.
only request parameter tuning.